pkg/rm/rm_remoting_test.go

```diff
@@ -0,0 +1,190 @@
+package rm
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/seata/seata-go/pkg/common"
+	"github.com/seata/seata-go/pkg/protocol/branch"
+	"github.com/seata/seata-go/pkg/tm"
+	"github.com/stretchr/testify/assert"
```

Comment:
thirdparty import should be above seata pkg

Reply:
solved